### DISCUSSION TITLE
Error when starting crazyswarm server

### DISCUSSION THREAD

**Discussion by GideonLombardo** [USC-ACTLab/crazyswarm#599] [Q&A] [answered]
Hello. I am almost done setting up crazyswarm. I am trying to start the crazyswarm server in order to run the helloworld script in real-life. 

When I try to start the crazyswarm server with: roslaunch crazyswarm hover_swarm.launch

I get a message:
Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

RLException: Invalid roslaunch XML syntax: not well-formed (invalid token): line 1, column 7
The traceback for the exception was written to the log file

I went to the log file to see if I could find any useful information. Unfortunately, the information in the log file was obscure. I have run the source ros_ws/devel/setup.bash, so I know that this is not the problem. Is there a specific file that I missed editing during the "configuration" stage? Please let me know.

**Comment by whoenig** [comment, marked as answer]
sounds like you made some invalid (?) edits to hover_swarm.launch. A git diff should tell you what you changed and you can post that, or the complete file here.

**Reply by GideonLombardo** [reply]
This has fixed the issue. Thank you so much.

Now, when I run python3 hello_world.py

I get an error message that says:  
ModuleNotFoundError: No module named 'crazyswarm'

**Reply by whoenig** [reply]
Sounds like you forgot to source your ROS workspace in that shell.

**Reply by GideonLombardo** [reply]
Yes, I got it working. Thank you so much.
